playbooks/prep-node-disks.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: all
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+  - name: run scan_disks
+    scan_disks:
+
+  - name: check to see what drives are mounted
+    shell: mount | grep {{ item }} | awk '{print $3}'
+    with_items: ssd_devices | union(hdd_devices)
+    ignore_errors: true
+    register: mount_point
+
+  - name: unmount all mounted devices
+    shell: umount {{ item.stdout }}; sed -i '\#{{ item.stdout }}#d' /etc/fstab
```

Comment:
There is a `mount` module: http://docs.ansible.com/ansible/mount_module.html  that may be useful here.